apps/backoffice-v2/src/pages/MerchantMonitoringBusinessReport/MerchantMonitoringBusinessReport.page.tsx

```diff
@@ -95,8 +100,51 @@ export const MerchantMonitoringBusinessReport: FunctionComponent = () => {
     onSubmit,
     deboardingReasonOptions,
     isFetchingBusinessReport,
+    locale,
   } = useMerchantMonitoringBusinessReportLogic();
 
+  // User should never really get in here, unless he manually sets the id in the URL.
+  // We don't want to prevent backend from sending data for reports that have not been completed yet,
+  // so instead we show a fallback UI.
+  if (
+    !isFetchingBusinessReport &&
+    businessReport?.status !== MERCHANT_REPORT_STATUSES_MAP['completed']
+  ) {
+    let supplementalText = '';
+
+    if (businessReport?.status === MERCHANT_REPORT_STATUSES_MAP['in-progress']) {
+      supplementalText = 'It is currently being processed by our system.';
+    } else if (businessReport?.status === MERCHANT_REPORT_STATUSES_MAP['quality-control']) {
+      supplementalText = 'It is currently being reviewed by our quality control team.';
+    }
```

Comment:
We don't want to tell our clients that a report is in quality control.

```suggestion
    if ([MERCHANT_REPORT_STATUSES_MAP['in-progress'], MERCHANT_REPORT_STATUSES_MAP['quality-control']].includes(businessReport?.status)) {
      supplementalText = 'It is currently being processed by our system.';
    }
```